client/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I don't think I've ever done anything with Cypress fixtures. Are they something useful that we should (eventually) be using?

Reply:
They are used to simulate API responses. I'm not too sure how useful they would be for us and I would be fine removing that example file if you want.

Reply:
No, feel free to leave it there. Or not – I really don't feel strongly about it at all. I should probably know more about how people use Cypress fixtures, but that's not for this PR. 😜